sql/changes/1.5/open_forms_callers.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+ALTER TABLE public.open_forms ALTER COLUMN id SET DEFAULT floor(random()*(1000000))+1;
+ALTER TABLE public.open_forms ADD COLUMN caller1 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller2 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller3 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller4 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN last_used timestamp without time zone;
```

Comment:
Reverted. When I start thinking about that, I was expecting this to have a longer life. Having the current opened form name could have been usefull for debugging or error logging.